### PR TITLE
Hide serials in exif display and extract

### PR DESCRIPTION
**Pull request by todtb**
#3527 

- Updated fixtures with exif meta data `:lens_serial_number`
- Refactored EXIF display for `Photo` by pulling into helper
- Excluding display serial number tags based on `exiftool`'s list of supported tags (see https://github.com/inaturalist/inaturalist/compare/main...todtb:3527-hide-serials-in-exif?expand=1#diff-14c62713d26766370b2cff9e80bcc20a6f53b555e3d657d873eddc3d546ff130R5)
- Prevent these tags from being extracted into `Photo`'s associated `metadata`

### CODE REVIEW

File: app/helpers/photos_helper.rb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    end.to_h
+  end
+
+  def visible_metadata( photo:, with_coords: )
+    return photo.metadata.select { _1.to_s =~ /copyright/i } unless logged_in?
+
+    photo.metadata.dup.tap do | md |
+      md.except!( *REJECTED_TAGS )
+      md.reject! { _1.to_s =~ /description/i }
+      md.reject! { _1.to_s =~ /(gps|date)/i } unless with_coords
```
Comment (kueda) [inline app/helpers/photos_helper.rb:39]: `_1` is a new piece of syntax for me, and it's something I think I've always wanted without knowing how to search for it. Is it actually documented anywhere?

File: app/helpers/photos_helper.rb [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module PhotosHelper
+  REJECTED_TAGS = ( ExifMetadata::REJECTED_TAGS + %i[dc dimensions] ).freeze
+
+  def metadata_table( photo )
```
Comment (kueda) [inline app/helpers/photos_helper.rb:6]: Can we call this `metadata_table_rows` since that's what it renders?

**Review by kueda — CHANGES_REQUESTED** [review]
Works for me, only one very minor change requested.

File: app/helpers/photos_helper.rb (hunk 2 above)
Comment (todtb) [reply app/helpers/photos_helper.rb:6]: Yeah, that makes sense 👍

**Review by kueda — APPROVED** [review]

**Pull request merged**